src/libraries/Microsoft.Extensions.Primitives/src/Microsoft.Extensions.Primitives.ref.g.cs

```diff
@@ -151,36 +156,37 @@ public partial struct Enumerator : System.Collections.Generic.IEnumerator<Micros
         public static bool operator ==(object? left, Microsoft.Extensions.Primitives.StringValues right) { throw null; }
         public static bool operator ==(string? left, Microsoft.Extensions.Primitives.StringValues right) { throw null; }
         public static bool operator ==(string?[]? left, Microsoft.Extensions.Primitives.StringValues right) { throw null; }
-        public static implicit operator string?(Microsoft.Extensions.Primitives.StringValues values) { throw null; }
-        public static implicit operator string?[]?(Microsoft.Extensions.Primitives.StringValues value) { throw null; }
-        public static implicit operator Microsoft.Extensions.Primitives.StringValues(string? value) { throw null; }
-        public static implicit operator Microsoft.Extensions.Primitives.StringValues(string?[]? values) { throw null; }
+        public static implicit operator string? (Microsoft.Extensions.Primitives.StringValues values) { throw null; }
+        public static implicit operator string?[]? (Microsoft.Extensions.Primitives.StringValues value) { throw null; }
+        public static implicit operator Microsoft.Extensions.Primitives.StringValues (string? value) { throw null; }
+        public static implicit operator Microsoft.Extensions.Primitives.StringValues (string?[]? values) { throw null; }
         public static bool operator !=(Microsoft.Extensions.Primitives.StringValues left, Microsoft.Extensions.Primitives.StringValues right) { throw null; }
         public static bool operator !=(Microsoft.Extensions.Primitives.StringValues left, object? right) { throw null; }
         public static bool operator !=(Microsoft.Extensions.Primitives.StringValues left, string? right) { throw null; }
         public static bool operator !=(Microsoft.Extensions.Primitives.StringValues left, string?[]? right) { throw null; }
         public static bool operator !=(object? left, Microsoft.Extensions.Primitives.StringValues right) { throw null; }
-        public static bool operator !=(string? left, Microsoft.Extensions.Primitives.StringValues right) { throw null; }
+        public static bool operator !=(string left, Microsoft.Extensions.Primitives.StringValues right) { throw null; }
```

Comment:
nullability change, cc @eerhardt

Reply:
This looks like a mistake in https://github.com/dotnet/runtime/pull/57395. The `==` operator has `string? left`. This one should as well. So I believe the fix here is to fix the `src` file.

cc @maxkoshevoi

Reply:
@maryamariyan - can you fix this one in #66215?